lib/presentation/crowd_action/pages/crowd_action_details.dart

```diff
@@ -524,16 +462,17 @@ class _CrowdActionDetailsPageState extends State<CrowdActionDetailsPage> {
                       height: 30,
                     ),
                     Text(
-                      widget.crowdAction.description,
+                      "You’re almost there! You’ve selected the displayed commitment to stick to through this CrowdAction. By clicking “Go!” you will officially commit to this goal.",
```

Comment:
If more than 1 commitment is selected, we should show
"the displayed commitments"
"officially commit to these goals"
Plural forms.